README.md

```diff
@@ -1,2 +1,22 @@
-# hw01
-Repository for homework 01
+# **Homework 1**
+## Biography ##
+***Basic Information***  
+
+Name:Ruixi(Ray) Mao  
+
+*  Current Status: third year grad student in Sociology 
```

Comment:
I think, if you want this to be a sub-list, you might want to add another tab before sub-list items

Reply:
I've got things fixed according to your advices. thx!